packages/core-php/src/TwigFunctions.php

```
@@ -21,6 +25,25 @@ public static function getSpacingScaleSequence() {
     ]);
   }
 
+
+  public static function inlineFile() {
+    return new Twig_SimpleFunction('inline', function($context, $filename) {
+      if (!$filename){
+        return '';
+      }
+
+      $data = new ArrayFinder($context['bolt']['data']);
+      $buildDir = $data->get('config.buildDir');
+
+      $fullPath = Path::join($buildDir, $filename);
+      if (file_exists($fullPath)){
+        return file_get_contents($fullPath);
+      }
```

What happens if the file doesn't exist? Let's make sure stuff breaks instead of silent errors. [Take a peak at what I do in TwigTools](https://github.com/basaltinc/twig-tools/blob/283904120c5d76443b8464e1c395a18644f55eda/src/Utils.php#L57) - I'm sure I have lots to learn on how to do this best, but this will fire an error for sure and prevent CI builds from happening and fire the watch notification errors at least.